pawn/Elements/AdministratorCommands.pwn

```diff
@@ -972,6 +972,16 @@ lvp_show(playerId, params[]) {
         showInfo = true;
         format(g_message, sizeof(g_message), "Save money and stats! Register your nickname at www.sa-mp.nl");
     }
+    
+    else if (!strcmp(showParameter, "guide", true, 5)) {
```

Comment:
I don't think this is particularly appropriate for `/show`, as very few people get temporary rights. There's no need to broadcast this to everyone on the server, is there?

Reply:
Yes sure .